MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -823,10 +828,31 @@ - (MXMemoryRoomStore*)getOrCreateRoomStore:(NSString*)roomId
     MXFileRoomStore *roomStore = roomStores[roomId];
     if (nil == roomStore)
     {
-        // MXFileStore requires MXFileRoomStore objets
-        roomStore = [[MXFileRoomStore alloc] init];
-        roomStores[roomId] = roomStore;
+        NSString *roomFile = [self messagesFileForRoom:roomId forBackup:NO];
+        if ([[NSFileManager defaultManager] fileExistsAtPath:roomFile])
+        {
+            @try
+            {
+                NSDate *startDate = [NSDate date];
+                roomStore = [NSKeyedUnarchiver unarchiveObjectWithFile:roomFile];
+                MXLogDebug(@"[MXFileStore] Loaded room messages of room: %@ in %.0fms, in main thread: %@", roomId, [[NSDate date] timeIntervalSinceDate:startDate] * 1000, [NSThread isMainThread] ? @"YES" : @"NO");
```

Comment:
To avoid too much noise with big account, can we log it only when we are not on the main thread?

The next game will be to avoid the display of this log.